### ISSUE TITLE
UI: Profile

### ISSUE THREAD

**Issue by kalienvivian** [NostrGit/NostrGit#213]
Hi.

I wonder if it would be possible for NostrGit to have a profile area similar to GitHub. would it be possible for NostrGit to have a profile area similar to GitHub? For example,

*something similar to this ui?*
<img src="https://user-images.githubusercontent.com/129428006/228972720-f0bb7fb6-01aa-4d91-b4d4-edd83af7b2b3.png" alt="image" width="240px"/>

**Comment by cypherhoodlum** [comment]
Yes, this is a planned feature. Users should at least be able to edit their nostr profile but we could include some other fields too, similar to GitHub. Maybe.

**Comment by cypherhoodlum** [comment]
Started to draft this UI to get #216 going.

**Comment by marcelolucario** [comment]
Hi cypherhoodlum, thanks for feedback, information and patience in reading and answering my question.
